### PR TITLE
feat: Implement GLPI portlet - EXO-67849

### PR DESCRIPTION
**Pull request by hakermi**
Implement GLPI portlet and no settings set ui message

### CODE REVIEW

File: glpi-integration-webapps/src/main/resources/locale/portlet/Glpi_en.properties
Comment (rdenarie) [inline glpi-integration-webapps/src/main/resources/locale/portlet/Glpi_en.properties:1]: Can you

- create the french file
- add the translation.properties file

File: glpi-integration-webapps/src/main/webapp/WEB-INF/gatein-resources.xml [hunk 1]
```diff
@@ -20,5 +20,28 @@
                   xsi:schemaLocation="http://www.exoplatform.org/xml/ns/gatein_resources_1_4 http://www.exoplatform.org/xml/ns/gatein_resources_1_4"
                   xmlns="http://www.exoplatform.org/xml/ns/gatein_resources_1_4">
 
-
+    <portlet>
+        <name>Glpi</name>
+        <module>
+            <script>
+                <minify>false</minify>
```
Comment (rdenarie) [inline glpi-integration-webapps/src/main/webapp/WEB-INF/gatein-resources.xml:27]: is it possible to minify ?

File: glpi-integration-webapps/src/main/webapp/vue-app/glpi/main.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import './initComponents.js';
+
+if (extensionRegistry) {
+  const components = extensionRegistry.loadComponents('Links');
+  if (components && components.length > 0) {
+    components.forEach(cmp => {
+      Vue.component(cmp.componentName, cmp.componentOptions);
+    });
+  }
+}
```
Comment (rdenarie) [inline glpi-integration-webapps/src/main/webapp/vue-app/glpi/main.js:28]: Is this needed ?

**Review by rdenarie — CHANGES_REQUESTED** [review]

File: glpi-integration-webapps/src/main/webapp/vue-app/glpi/main.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+import './initComponents.js';
+
+if (extensionRegistry) {
+  const components = extensionRegistry.loadComponents('Glpi');
+  if (components && components.length > 0) {
+    components.forEach(cmp => {
+      Vue.component(cmp.componentName, cmp.componentOptions);
+    });
+  }
+}
```
Comment (rdenarie) [inline glpi-integration-webapps/src/main/webapp/vue-app/glpi/main.js:28]: Why do you have this ? do we have component registred in extensionRegistry for glpi integration ?

**Review by rdenarie — CHANGES_REQUESTED** [review]

File: glpi-integration-webapps/src/main/webapp/WEB-INF/gatein-resources.xml (hunk 1 above)
Comment (hakermi) [reply glpi-integration-webapps/src/main/webapp/WEB-INF/gatein-resources.xml:27]: No, we can't minify, all portlets should not be minified, so the eva

File: glpi-integration-webapps/src/main/resources/locale/portlet/Glpi_en.properties
Comment (hakermi) [reply glpi-integration-webapps/src/main/resources/locale/portlet/Glpi_en.properties:1]: added, the french locale file, the translations.properties already exists

File: glpi-integration-webapps/src/main/webapp/WEB-INF/gatein-resources.xml (hunk 1 above)
Comment (rdenarie) [reply glpi-integration-webapps/src/main/webapp/WEB-INF/gatein-resources.xml:27]: I think you cut your answer

Comment (hakermi) [reply glpi-integration-webapps/src/main/webapp/WEB-INF/gatein-resources.xml:27]: I wanted to say that if we minify we will have issues with the eval of the script, and as result we will have errors and the portlet will not be initialized

File: glpi-integration-webapps/src/main/webapp/vue-app/glpi/main.js (hunk 3 above)
Comment (rdenarie) [reply glpi-integration-webapps/src/main/webapp/vue-app/glpi/main.js:28]: so to remove. We will add this when/if needed

Comment (hakermi) [reply glpi-integration-webapps/src/main/webapp/vue-app/glpi/main.js]: removed

**Pull request merged**